posthog/migrations/0155_auto_20210520_1756.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.1.8 on 2021-05-20 17:56
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Nit for next time, worth giving this file a more recognizable name

Reply:
ohh I didn't know we could override the migration names, good to know 🧠